bundles/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoBinding.java

```diff
@@ -253,8 +252,8 @@ public void updated(Dictionary<String, ?> config) throws ConfigurationException
                 Matcher matcher = EXTRACT_CONFIG_PATTERN.matcher(key);
 
                 if (!matcher.matches()) {
-                    logger.debug(
-                            "given config key '" + key + "' does not follow the expected pattern '<id>.<host|port>'");
+                    logger.debug("given config key '" + key
```

Comment:
Definitely use parameterized logging here instead of string concatenation, because otherwise this method will be generating garbage objects on every call!